src/components/sections/TheNewArrivals.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <section class="arrival spad">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 text-center">
+          <div class="section-title">
+            <span>Hoa tươi</span>
+            <h2>Mới Nhập Về</h2>
+          </div>
+          <div class="filter__controls">
+            <ul>
+              <li
+                v-for="item in filterSet"
+                :key="item.id"
+                :class="{ active: selectedFilterId == item.id }"
+                @click="selectedFilterId = item.id"
+              >
+                {{ item.filterName }}
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <template v-for="item in newArrivalsItems">
+          <div
+            v-show="selectedFilterId == 0 || item.filterId == selectedFilterId"
+            :key="item.productId"
+            class="d-col"
+          >
+            <div class="product__item">
+              <div
+                :style="{
+                  'background-image': 'url(' + item.bgImgUrl + ')',
+                }"
+                class="product__item__pic set-bg"
+              >
+                <div v-if="item.label != ''" class="label">
+                  {{ item.label }}
+                </div>
+                <ul class="product__item__hover">
+                  <li>
+                    <!-- Button trigger Product Preview Modal -->
+                    <button
+                      type="button"
+                      data-bs-toggle="modal"
+                      data-bs-target="#productPreviewModal"
+                      @click="quickViewProductId = item.productId"
+                    >
+                      <span
+                        data-bs-toggle="tooltip"
+                        data-bs-placement="bottom"
+                        title="Xem chi tiết"
+                        ><BaseIconSearch
+                      /></span>
+                    </button>
+                  </li>
+                  <li>
+                    <button
+                      type="button"
+                      data-bs-toggle="tooltip"
+                      data-bs-placement="bottom"
+                      :title="
+                        item.quantity == 0 ||
+                        item.quantityInCart == item.quantity
+                          ? 'Hết số lượng để thêm vào giỏ'
```

Comment:
Nếu đã hết không thể thêm vào giỏ thì anh nên disable nút đi. Chứ đổi mỗi text như này thì họ vẫn click dc mà nhỉ

Reply:
[Link to commit fix .vue file](https://github.com/binhbdn/flower-shop/pull/20/commits/4a3e6771fe264f45b91bc421b55face51b2ca353
[Link to commit fix .scss file](https://github.com/binhbdn/flower-shop/pull/20/commits/66b659b8212618fbc531954cfbacc2a60285b487)